ivysettings.xml

```diff
@@ -33,7 +33,7 @@
         <ivy pattern="${localMaveRepoDir}/[organisation]/[module]/[revision]/[module]-[revision].pom" />
       </filesystem>
       -->
-      <ibiblio name="mavenCentral" m2compatible="true" />
+      <ibiblio name="mavenCentral" m2compatible="true" root="https://nexus.intern/repository/central/" />
```

Comment:
what is this root="https://nexus.intern/repository/central/"? it feels like a personal setup: is it really expected to be shared?